src/main/java/org/spongepowered/api/event/message/MessageSinkEvent.java

```diff
@@ -47,14 +49,14 @@
      *
      * @return The message sink the message in this event will be sent to
      */
-    MessageSink getSink();
+    Optional<MessageSink> getSink();
```

Comment:
Using `Optional<MessageSink>` prevents any processing, whereas `MessageSinks.NONE` would still process for each recipient.